src/entities/Message/Message.controller.ts

```diff
@@ -0,0 +1,27 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response } from 'express';
+import validateMessageReq from './Message.validation';
+import Message from '../../entities/Message/Message.schema'
+
+const { BAD_REQUEST, CREATED } = StatusCodes;
+
+export const sendMessage = async (req: Request, res: Response) => {
+    const { error } = validateMessageReq(req.body);
+    if (error) return res.status(BAD_REQUEST).send(error.details[0].message);
+    if (req.body.recipient === req.user) {
+        return res.status(BAD_REQUEST).send("You can't send a message to yourself")  
+    } 
```

Comment:
Doesn't work if you try send message to yourself. Although it seems like it should work 🙄 

Works with this:
```suggestion
    const sender = await User.findById(req.user);
    if (sender) { 
        if (req.body.recipient == sender._id) {
            return res.status(BAD_REQUEST).send("You can't send a message to yourself")  
        } 
    }
```